src/valora-dapp-list.json

```diff
@@ -42,6 +42,12 @@
     }
   ],
   "applications": [
+    {
```

Comment:
We should likely add this to the end of the Dapp list - along with Staked Celo so that new Dapps aren't listed above existing Dapps. @jeanregisser we don't have documentation about what is expected here.

Reply:
@MuckT thanks for your feedback. I wasn't sure whether it was my call to also add `Staked Celo` to the bottom, but as per your comment I did that.